src/views/router_views/individual_statement/individual_statement.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block main_content %}
+
```

Comment:
Do we want to add `[name]` above the title as per the prototype?  I know it'll have to be done properly, but just as a placeholder?

Reply:
Yes :)

Reply:
I'm struggling to see a GDS component that would place the name above the title.  The checkbox component places a hint below the heading.
Either we mention this to UX or I guess we can just use plain html with a `govuk-caption-xl` class ???

Reply:
Got this working locally.

Reply:
It's below the heading though.. see my comment above

Reply:
the ts file formatting is slightly off, and the actual elements and classes may change later, but meets what we need now.